modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/sequence/HeadersSideBar.scala

```diff
@@ -1,28 +1,37 @@
 package edu.gemini.seqexec.web.client.components.sequence
 
+import diode.react.ModelProxy
+
 import edu.gemini.seqexec.web.client.components.DropdownMenu
-import japgolly.scalajs.react.ReactComponentB
+import edu.gemini.seqexec.web.client.semanticui.elements.label.Label
+import edu.gemini.seqexec.web.client.semanticui.elements.input.Input
+import edu.gemini.seqexec.web.client.model._
+
+import japgolly.scalajs.react.{ReactComponentB, Callback}
 import japgolly.scalajs.react.vdom.prefix_<^._
 
 /**
   * Display to show headers per sequence
   */
 object HeadersSideBar {
-  val component = ReactComponentB[Unit]("HeadersSideBar")
+  case class Props(operator: ModelProxy[Option[String]], status: ModelProxy[ClientStatus])
+
+  val component = ReactComponentB[Props]("HeadersSideBar")
     .stateless
-    .render(_ =>
+    .render_P(p =>
       <.div(
         ^.cls := "ui raised secondary segment",
         <.h4("Headers"),
         <.div(
           ^.cls := "ui form",
           <.div(
             ^.cls := "required field",
-            <.label("Operator"),
-            <.input(
-              ^.`type` :="text",
-              ^.autoComplete :="off"
-            )
+            Label(Label.Props("Operator", "operator")),
+            Input(Input.Props("operator", "operator",
+              p.operator().getOrElse(""),
+              placeholder = "Operator...",
+              disabled = !p.status().isLogged,
+              onBlur = name => p.operator.dispatchCB(UpdateOperator(name))))
```

Comment:
Good question. `onBlur` is a javascript event that you get when the element looses focus. This is not totally optimal and some extra work is needed to get it working at all times. You'd think that we should use `onChange` which is triggered every time the user presses something but that causes undesirable UI artifacts as you'd update the frontend every time the users presses a key